### PR TITLE
Order apps

### PR DESCRIPTION
**Pull request by pat-s**
Alphabetically, case-insensitive

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,8 +187,9 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [Awareness](http://iamfutureproof.com/tools/awareness/)
 - [AWS Command Line Interface](https://aws.amazon.com/cli/)
 - [Bartender](http://www.macbartender.com/)
-- [Bash it](https://github.com/Bash-it/bash-it)
 - [Bash](http://www.gnu.org/software/bash/)
+- [Bash it](https://github.com/Bash-it/bash-it)
+- [BBEdit](http://www.barebones.com/products/bbedit/)
```
Comment (lra) [inline README.md]: I think you are using an outdated list, it was not in `master`.

File: README.md [hunk 2]
```diff
@@ -266,8 +267,8 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [fastlane](https://fastlane.tools)
 - [Feeds](http://www.feedsapp.com/)
 - [FileZilla](https://filezilla-project.org/)
-- [Fish](http://ridiculousfish.com/shell/)
 - [Fisherman](http://fisherman.sh/)
+- [Fish](http://ridiculousfish.com/shell/)
```
Comment (lra) [inline README.md]: Up there, you put `Bash` before `Bash it`, but here you put `Fish` after `Fisherman`. Let's be consistent ;)

File: README.md [hunk 3]
```diff
@@ -281,9 +282,9 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [GeekTool](http://projects.tynsoe.org/en/geektool/)
 - [GHCi](https://wiki.haskell.org/GHC/GHCi)
 - [Ghostwriter](https://wereturtle.github.io/ghostwriter/)
+- [Gitbox](http://gitboxapp.com/)
```
Comment (lra) [inline README.md]: ditto

File: README.md [hunk 4]
```diff
@@ -299,8 +300,8 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [HandBrake](https://handbrake.fr/)
 - [Hands Off!](http://www.oneperiodic.com/products/handsoff/)
 - [Hazel](http://www.noodlesoft.com/hazel.php)
-- [Hero Lab](http://www.wolflair.com/index.php?context=hero_lab)
 - [Heroku](https://www.heroku.com/)
+- [Hero Lab](http://www.wolflair.com/index.php?context=hero_lab)
```
Comment (lra) [inline README.md]: ditto, space is before `k`, no?

File: README.md [hunk 5]
```diff
@@ -429,42 +431,42 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [Popcorn-Time](https://popcorntime.io/)
 - [PostgreSQL](http://www.postgresql.org/)
 - [Postico](https://eggerapps.at/postico/)
-- [Pow](http://pow.cx/)
 - [Powerline-shell](https://github.com/b-ryan/powerline-shell)
+- [Pow](http://pow.cx/)
```
Comment (lra) [inline README.md]: ditto

File: README.md [hunk 6]
```diff
@@ -429,42 +431,42 @@ See the [README](doc/README.md) file in the doc directory for more info.
...
 - [Querious](http://www.araelium.com/querious/)
 - [Quicksilver](http://qsapp.com/)
 - [Qutebrowser](http://qutebrowser.org/)
-- [R](http://www.r-project.org/)
 - [Rails](http://rubyonrails.org/)
 - [Ranger](https://ranger.github.io/)
-- [Redshift Scheduler](https://github.com/spantaleev/redshift-scheduler)
 - [Redshift](http://jonls.dk/redshift/)
+- [Redshift Scheduler](https://github.com/spantaleev/redshift-scheduler)
+- [R](http://www.r-project.org/)
```
Comment (lra) [inline README.md]: R should be 1st, not last

File: README.md [hunk 7]
```diff
@@ -429,42 +431,42 @@ See the [README](doc/README.md) file in the doc directory for more info.
...
 - [Robomongo](http://robomongo.org/)
 - [Royal TSX](http://www.royaltsx.com/ts/osx/features)
 - [RStudio](https://www.rstudio.com/)
 - [rTorrent](http://libtorrent.rakshasa.no/)
 - [Rubocop](https://github.com/bbatsov/rubocop)
-- [Ruby Version Manager](https://rvm.io/)
-- [Ruby Version](https://gist.github.com/fnichol/1912050)
 - [Ruby](https://www.ruby-lang.org/)
 - [RubyMine](http://www.jetbrains.com/ruby/)
+- [Ruby Version](https://gist.github.com/fnichol/1912050)
```
Comment (lra) [inline README.md]: `[space]` before any letter

File: README.md [hunk 8]
```diff
@@ -429,42 +431,42 @@ See the [README](doc/README.md) file in the doc directory for more info.
...
+- [Ruby Version Manager](https://rvm.io/)
```
Comment (lra) [inline README.md]: ditto

File: README.md [hunk 9]
```diff
@@ -429,42 +431,42 @@ See the [README](doc/README.md) file in the doc directory for more info.
...
 - [S3cmd](http://s3tools.org/s3cmd)
 - [SABnzbd](http://sabnzbd.org/)
 - [SBCL](http://www.sbcl.org/)
 - [SBT](http://www.scala-sbt.org/)
 - [Scenario](http://www.lagentesoft.com/scenario/)
-- [Screen](http://www.gnu.org/software/screen/)
 - [Screenhero](https://screenhero.com)
+- [Screen](http://www.gnu.org/software/screen/)
```
Comment (lra) [inline README.md]: ditto

File: README.md [hunk 10]
```diff
@@ -513,12 +514,14 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [TextExpander](https://smilesoftware.com/textexpander)
 - [TextMate](http://macromates.com/)
 - [Textual](http://www.codeux.com/textual/)
+- [T](http://sferik.github.io/t/)
```
Comment (lra) [inline README.md]: Why is no letter between `e` and `i`?

File: README.md [hunk 11]
```diff
@@ -529,14 +532,15 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [Twitterrific](http://twitterrific.com/)
 - [Typinator](http://www.ergonis.com/products/typinator/)
 - [Typora](https://typora.io)
+- [Uebersicht](http://tracesof.net/uebersicht/)
```
Comment (lra) [inline README.md]: You changed `Ü` to `U`.

**Comment by pat-s** [comment]
I did it in vim using `%!sort -fi`. Didn't double check all entries therefore - will clean up tmr :+1:

**Comment by pat-s** [comment]
Hm, all my automatic vim attempts seem to have some inconsistencies in it. Resolving by hand is problematic as it would introduce them again when updating in some weeks/months. 

Can it be so hard to sort a markdown list of links? Apparently, yes :laughing: 	:sweat_smile:

**Pull request closed without merging**